test/test_file.py

```diff
@@ -0,0 +1,45 @@
+# This file represents all known cases of special-case indentations
```

Comment:
I think it was mainly hanging indent failures. They're Jasmine tests, which
are rather simple and easy to get started with. I only wrote my first a few
months ago.

Typically, you don't want too many outside file dependencies for a unit
test, but for this case, I wouldn't see a problem with loading it into an
array split on lines, then test using that.